lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1506,7 +1571,8 @@ class ConvertAtenDetachOp : public OpConversionPattern<AtenDetachOp> {
       return failure();
 
     Type resultType = getTypeConverter()->convertType(op.getType());
-    rewriter.replaceOpWithNewOp<tensor::CastOp>(op, resultType, adaptor.getSelf());
+    rewriter.replaceOpWithNewOp<tensor::CastOp>(op, resultType,
```

Comment:
Unrelated reformatting?

Reply:
Hm, my "format changed lines" in vscode doesn't seem to always work. Sorry for missing that before review.